.env-default

```diff
@@ -0,0 +1,6 @@
+SUPPORT_URL=http://localhost:3000
+SUPPORT_PUBLIC_KEY=BTS4wDJjaKzywGU9RaQpx1JvjFnnWd4pibP9hxYdphXHUdt2wvqAS
+API_COINS_URL=https://api.testnet.crypto-bridge.org/api/v1/coins
+API_TRADING_PAIRS_URL=https://api.testnet.crypto-bridge.org/api/v1/trading-pairs
+FRESHDESK_API_KEY=htHwvbG7s5oYh0UDhF
```

Comment:
I do not think we have a possibillity of having a read-only key. This is for accessing the FAQ on FreshDesk, but this will eventually be removed, as all FAQs will move to JIRA.